### PR TITLE
Add initial helm community documentation

### PR DESCRIPTION
**Pull request by fabianvf**

### CODE REVIEW

**Review by asmacdo — APPROVED** [review]
/lgtm

**Review by jmrodri — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -82,3 +82,22 @@ To organize and steer the development of the [Operator SDK](https://github.com/o
 
 - Slack: [##operator-sdk-dev](https://kubernetes.slack.com/messages/operator-sdk-dev)
 - [Mailing list](https://groups.google.com/forum/#!forum/operator-framework)
+
+## Helm Operator Plugins Working Group
+
+To organize and steer the development of [helm-operator-plugins](https://github.com/operator-framework/helm-operator-plugins) for the SDK, and related projects.
```
Comment (asmacdo) [inline README.md:88]: optional nit 
```suggestion
To organize and steer the development of [helm-operator-plugins](https://github.com/operator-framework/helm-operator-plugins) and related projects.
```

**Pull request merged**